src/frontend/src/pages/new-challenge/new-challenge.tsx

```diff
@@ -1,7 +1,150 @@
-import React from 'react';
+import React from "react";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import SubmitChallengePopup from "./components/submit-popup";
+import {
+    Grid,
+    Button,
+    Card,
+    CardContent,
+    Typography,
+    CardActions,
+    Container,
+    Box,
+    TextField
+} from "@material-ui/core";
 
 export const NewChallenge: React.FC = () => {
     return (
-        <div>New Challenge Page</div>
-    )
+        <div>
+            {/*<div style={{ padding: 50 }}>*/}
+            <Box m={5}>
+                <Container>
+                    {/*<Box p={10}>*/}
+                    <Box p={2}>
```

Comment:
```suggestion
            <Container>
                <Box p={5}>
```
You should make the spacing inside the container rather than outside. That means you don't need spacing around each of the elements in your container. Also, the Typography component already creates a block level element so it isn't necessary to surround it with a Box.